### PR TITLE
various little fixes

### PR DESCRIPTION
**Pull request by jnchin314**
readme updates

a little code org

settings to be more compatible out of the repo to work with macosx and unix

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,5 @@ venv.bak/
 !.vscode/launch.json 
 !.vscode/extensions.json 
 .history
+.idea/*
```
Comment (jnchin314) [inline .gitignore:30]: jetbrains IDE pycharm stuff

postgresDB was installed in this folder, woops!  too lazy to move my postgres or reinit it somewhere else

File: leaderboards/views.py [hunk 2]
```diff
@@ -60,11 +58,14 @@ def player_search(request):
     }
     return render(request, 'leaderboards/player_search.html', context)
 
+
+#Is this used?  Can we remove?
```
Comment (jnchin314) [inline leaderboards/views.py:62]: if we can remove that would be great but i wasn't sure if you were using these or not...

File: leaderboards/views.py [hunk 3]
```diff
@@ -179,8 +176,6 @@ def player_profile(request, playfabid):
     leaderboard_data = []
     latest_serial_numbers = {}
     for leaderboard_name in leaderboards:
-        print("Looking for:", leaderboard_name)
```
Comment (jnchin314) [inline leaderboards/views.py:182]: so many logs

File: chivcreds.txt
Comment (jnchin314) [inline chivcreds.txt:1]: added this by mistake yesterday

**Review by gimmic — APPROVED** [review]

**Pull request merged**